lib/notify/notification.rb

```diff
@@ -0,0 +1,39 @@
+require 'notifications/client'
+
+module Notification
+  INVITE_TEMPLATE = "afdb4827-0f71-4588-b35d-80bd514f5bdb".freeze
+
+  def mail_client
+    check_for_key
+    Notifications::Client.new(Rails.configuration.notify_key)
+  end
+
+  def send_invitation_email(opts)
+    mail_client.send_email(
+      email_address: opts[:email_address],
+      template_id: INVITE_TEMPLATE,
+      personalisation: {
+        first_name: opts[:first_name],
+        url: url,
+        temporary_password: opts[:temporary_password],
+      },
+     )
+  end
+
+private
+
+  def check_for_key
+    if Rails.configuration.notify_key.nil?
```

Comment:
since the value is being loaded in the initializer, we probably don't need to check here again...

Reply:
Without the key it doesn't give the most obvious error:

<img width="603" alt="Screen Shot 2019-12-02 at 12 18 48" src="https://user-images.githubusercontent.com/5963488/69959022-0963b700-14fe-11ea-873b-81ec79428b5d.png">

Reply:
ah, it's because the dev is not using the fetch function, I assumed they were all the same... I would suggest changing `config.notify_key = ENV['TEST_NOTIFY_KEY']` in development.rb to using fetch function. It'll then fail there and hopefully the error message is more useful?

Reply:
thanks, have changed. The reason I changed it to use `[]` instead for `.fetch` was so that the app could still be loaded without the keys... just people wouldn't be able to send emails... but I guess it is safer just to make sure we use them.

Reply:
yeah, they could just add a dummy value to the .env for startup...